src/resources/dicts/fr.edn

```diff
@@ -172,7 +172,23 @@
     :linked-references/filter-excludes "Exclut : "
     :linked-references/filter-heading "Filtrer"
     :linked-references/filter-includes "Inclut : "
-    :linked-references/reference-count (fn [filtered-count total] (str filtered-count (when filtered-count (if (= filtered-count 1) " référence liée" " références liées")) " parmi " total))
+    :linked-references/reference-count (fn [filtered-count total]
```

Comment:
Also worth reverting

Reply:
Also the current translation is not correct: if filtered-count is nil, the translation is "parmi total" ("among total" which is too short to be clear).
Do you have another solution to make plural translations which handle all cases with short syntax without `cond`?
Thanks

Reply:
Same as other comment

Reply:
Also simplified the translation, removing the case when `total` is 0.

Do you agree with the changes?